gdx/src/com/badlogic/gdx/math/Interpolation.java

```diff
@@ -294,14 +294,14 @@ public Bounce (int bounces) {
 		}
 
 		private float out (float a) {

```

Comment:
^You forgot to change this parameter.

Reply:
Thanks, it's fixed!

I was meant to forget one since I've done those edits in GitHub directly.